### PR TITLE
gh-125584: Require network resource in test_urllib2.HandlerTests.test_ftp_error

### PR DESCRIPTION
**Pull request by mgorny**
* Issue: gh-125584

### CODE REVIEW

File: Misc/NEWS.d/next/Tests/2024-10-16-14-33-57.gh-issue-125584.5goH3R.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Added the missing requirement of ``network`` resource for
+``test.test_urllib2.HandlerTests.test_ftp_error``.
```
Comment (Eclips4) [inline Misc/NEWS.d/next/Tests/2024-10-16-14-33-57.gh-issue-125584.5goH3R.rst]: We can omit the `NEWS` entry from this PR. This only affects tests, and there is only one release with that bug, which is `3.14a1`. So, please, remove it.

Comment (mgorny) [reply Misc/NEWS.d/next/Tests/2024-10-16-14-33-57.gh-issue-125584.5goH3R.rst]: Sure, removed.

**Review by Eclips4 — APPROVED** [review]
Thanks!

**Comment by mgorny** [comment]
Thanks!

**Pull request merged**